### DISCUSSION TITLE
2021 Comprehensive Workflow video script?

### DISCUSSION THREAD

**Discussion by ShreyaPatibanda** [BryanJenksCommunity/FAQ#192] [Q&A]
I was wondering if the markdown docs you display in the 2021 Comprehensive Workflow video is open for the public domain. Please link it if is thanks. I would love to copy and paste these notes and modify them in my own vault.

**Comment by tallguyjenks** [comment]
i dont think i offer them as part of my template vault but no one has ever asked before. maybe for my sponsors
